### DISCUSSION TITLE
Cannot "Add Blog" or "Import Blog List" with valid URL

### DISCUSSION THREAD

**Discussion by NadiaThePuggle** [TumblThreeApp/TumblThree#373] [Q&A (Question & Answer)]
**Description**
After loading the TumblThree application, the only buttons on the bottom of the user interface that are active and working are as follows:

- Import Blog List
- Download
- Settings
- About

The Check Clipboard is active, but does nothing when I click on it.

**Steps**
1. I authenticate to my tumblr account in Settings.  

2. I try entering URL the "Enter URL:" text input box in the following valid formats according to your documentation and discussion forums where <blog_name> is a valid blog that I can access through the tumblr UI:

- https://www.tumblr.com/<blog_name>/
- https://www.tumblr.com/<blog_name>
- "https://www.tumblr.com/<blog_name>/"
- "https://www.tumblr.com/<blog_name"
- www.tumblr.com/<blog_name>/
- www.tumblr.com/<blog_name>
- "www.tumblr.com/<blog_name>/"
- "www.tumblr.com/<blog_name>"

No error, no nothing.

3. I enter the blog URL in a blogs.txt file trying the same formats as above and import the blogs.txt file using the Import Blog List button.  This time I do get an error message at the top of the UI:

`Error 1: Could not add the blog: The url https://www.tumblr.blog/<blog_name>/ cannot be recognized as valid blog.`

**Desktop (please complete the following information):**
 - TumblThree version: 2.8.3
 - OS: Windows 10 Professional
 - Browser: Brave
 - Version:  1.44.112 Chromium: 106.0.5249.119 (Official Build) (64-bit)](https://brave.com/latest/)

**Comment by thomas694** [comment]
Please enter the URL in the format `<blogname>.tumblr.com`
The `Check Clipboard` activates/deactivates the functionality to check the clipboard for URLs in the format just mentioned.

**Comment by NadiaThePuggle** [comment]
Thank you for the prompt response!

**Reply by Technotron21** [reply]
I seem to be running into a similar issue suddenly, since nearly every blog now gives me the error of "Online check for 'blog' failed: Invalid URI: The format of the URI could not be determined." Not sure how to fix it, since the one blog that doesn't have it is formatted like all the others.

EDIT: Okay, must've been a VPN thing, cause now it's fixed itself after exiting. Weird that it happened still.
